items.json

```diff
@@ -124221,11 +124221,11 @@
         "itemUsage": "shield",
         "accuracy": 100,
         "missileSpeed": 0,
-        "stackAmount": 600,
-        "length": 118,
+        "stackAmount": 383,
+        "length": 87,
         "balance": 0.0,
         "handling": 80,
```

Comment:
Why are you changing the Length? The length needs to exactly match the model size? Did this length change come from Meow? Otherwise this is just cheating by claiming it smaller in order to get more stat for free.

Reply:
If the goal is to lower the weight , reduce health further and increase armor

Reply:
Diff fucked up by alphabetical order change